src/foremast/datapipeline/__main__.py

```diff
@@ -0,0 +1,55 @@
+#   Foremast - Pipeline Tooling
+#
+#   Copyright 2016 Gogo, LLC
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""CLI entry point for aws datapipeline creation.
+
+Help: ``python -m src.foremast.datapipeline -h``
+"""
+import argparse
+import logging
+
+from ..args import add_app, add_debug, add_env, add_properties, add_region
+from ..consts import LOGGING_FORMAT
+
+from .datapipeline import AWSDataPipeline
+
+
+def main():
+    """Create datapipeline."""
+    logging.basicConfig(format=LOGGING_FORMAT)
+    log = logging.getLogger(__name__)
+
+    parser = argparse.ArgumentParser(description=main.__doc__)
+    add_debug(parser)
+    add_app(parser)
+    add_env(parser)
+    add_properties(parser)
+    add_region(parser)
+    args = parser.parse_args()
+
+    logging.getLogger(__package__.split('.')[0]).setLevel(args.debug)
+
+    log.debug('Parsed arguments: %s', args)
+
+    datapipeline = AWSDataPipeline(app=args.app,
```

Comment:
Yeah, but all the other Modules have drifted away from what we're doing in `foremast.runner`. We were discussing previously about getting rid of the `__main__.py` files because of that issue. If we can call `foremast.runner` like we do now, we can start to at least make the `__main__.py` files relevant in case someone is using them.